src/View/Helper/ListsHelper.php

```diff
@@ -558,10 +608,21 @@ public function displayListsModule($listsArray)
                         $list['id']
                     )
                 );
+                if(currentUser::isMember()){
+                    if(($list['user_id'] == $currentUserId && $list['editable_by'] != 'no_one') || $list['editable_by'] == 'anyone')
+                    {
+                       echo $this->_displayRemoveLink($list['id'], $sentenceId, $list['name']);
+                    }                   
+                }
                 echo '</li>';
             }
             echo '</ul>';
             echo '</div>';
+        } else {
+            echo '<div class="section md-whiteframe-1dp">';
+            /* @translators: header text on the sidebar of a sentence page */
+            echo $this->Html->tag('h2', __('Lists'));
+            echo '</div>';
```

Comment:
These 4 lines are always executed on the other side of the if branch too, so you can take them out of this if-else block.